merlin/celery.py

```diff
@@ -97,15 +100,15 @@
 
 # load config overrides from app.yaml
 if (
-    (not hasattr(CONFIG.celery, "override"))
+    not hasattr(CONFIG.celery, "override")
     or (CONFIG.celery.override is None)
-    or (len(nested_namespace_to_dicts(CONFIG.celery.override)) == 0)
+    or (not nested_namespace_to_dicts(CONFIG.celery.override))  # only true if len == 0
 ):
     LOG.debug("Skipping celery config override; 'celery.override' field is empty.")
 else:
-    override_dict = nested_namespace_to_dicts(CONFIG.celery.override)
-    override_str = ""
-    i = 0
+    override_dict: Dict = nested_namespace_to_dicts(CONFIG.celery.override)
+    override_str: str = ""
+    i: int = 0
     for k, v in override_dict.items():
```

Comment:
Ideally these would have some sort of forward declaration, but I have no idea what they would be.